### PR TITLE
Sidebar code

### PR DESCRIPTION
**Pull request by SteveOye**
Clubhouse: https://app.clubhouse.io/startng/story/49692/sidebar-code

Video: https://drive.google.com/file/d/1QlTXAIluWG4zdiEmjTzLaHKiwYALosr6/view?usp=sharing

### CODE REVIEW

**Comment by allenbangai** [comment]
this file is not ok..

**Comment by allenbangai** [comment]
video is correct but this file is not ok 
image

File: app/src/main/java/com/michael/afrivac/SplashscreenActivity.java [hunk 1]
```diff
@@ -28,18 +28,17 @@ protected void onCreate(Bundle savedInstanceState) {
             public void run() {
 
                 if (sharePref.getIsUserLoggedIn()) {
-                    startActivity(new Intent(getApplicationContext(), LoginActivity.class));
+                    startActivity(new Intent(getApplicationContext(), MainActivity.class));
                 } else {
-                    Intent intent = new Intent(getApplicationContext(), OnboardscreenActivity.class);
+                    Intent intent = new Intent(getApplicationContext(), MainActivity.class);
```
Comment (allenbangai) [inline app/src/main/java/com/michael/afrivac/SplashscreenActivity.java:33]: wrong, from your plash screen, the user is suppose to go to onboardgin then login activty, sign it or sign up before accessing main activity... your video is suppose to show that process.. Also, you had conflicting files and did not clone the repo yesterdy beore doing your update... pls clone and do your update... run the app and make sure things are ok.. I don't want to be fixing issues every night...

feed back to me as fast as possible...

**Comment by allenbangai** [comment]
pls do not merge this ..

**Comment by SteveOye** [comment]
@allenbangai @Michaelzy27 check it out now

**Pull request merged**